spot_data_responses.go

```diff
@@ -104,6 +104,36 @@ type SingleMarketStatistics struct {
 	Data TickerData
 }
 
+// AllMarketStatistics
+type AllMarketStatistics struct {
+	GeneralResponse
+	Data TickerData
+}
+
+// CurrencyRate
+type CurrencyRate struct {
+	GeneralResponse
+	Data TickerData
```

Comment:
Define a custom type instead of wrongly using TickerData everywhere,
we've spoken about it in detail earlier.